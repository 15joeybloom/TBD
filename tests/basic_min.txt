select min(majorid) from student
