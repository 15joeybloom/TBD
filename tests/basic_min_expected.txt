
SQL> 
min(majorid)
------
    10

SQL> 
